sqlglot/generator.py

```diff
@@ -79,11 +79,13 @@ def __init__(
         unsupported_level=ErrorLevel.WARN,
     ):
         # pylint: disable=too-many-arguments
+        import sqlglot
```

Comment:
I'm assuming we need to do this because it's a circular import?

Reply:
Yep that is correct

Reply:
Opinions on moving it to another module? Like... `sqlglot.settings`?

I know sqlglot already has a bunch of circular import workarounds like this... but 🤷 . Seems like something we should try to avoid.

Reply:
Yeah I actually initially had a version of this where I created a `config` file so it was `sqlglot.config.pretty`. This approach is really simple when using it from Minerva since you just do `sqlglot.pretty = True`. I'm open either way.

Reply:
`sqlglot.config.pretty = True` doesn't feel too different to me. But I don't have a strong opinion here.

Reply:
let’s just keep it as is, very simple